ReactiveCocoa/Reusable.swift

```diff
@@ -0,0 +1,10 @@
+import ReactiveSwift
+import enum Result.NoError
+
+public protocol Reusable: class {}
```

Comment:
This is extremely fragile though because `extension Reactive where Base: NSObject, Base: Reusable` does not guarantee that `Base` has a `prepareForReuse` method 👎

Where is that error exactly?